package.json

```diff
@@ -26,9 +26,11 @@
     "electron-build": "electron-builder --mac --windows --linux",
     "electron-dev": "concurrently \"BROWSER=none yarn start\" \"wait-on http://localhost:3000 && electron .\"",
     "format:staged": "lint-staged",
+    "generate-types": "yarn generate-types:contracts",
```

Comment:
perhaps, we can make use of this lib https://www.npmjs.com/package/npm-run-all

Reply:
yeah looks nice, but lets leave it when we get to adding other types